### PR TITLE
[Nightly] Pass args through to executable

### PR DESCRIPTION
**Pull request by ajpal**
Follow up to https://github.com/egraphs-good/eggcc/pull/417, fixes the segfaults by passing args through

### CODE REVIEW

File: src/util.rs [hunk 1]
```diff
@@ -660,11 +667,17 @@ impl Run {
...
             .unwrap();
 
-        if self.in_test && !self.interp {
+        if self.in_test {
             std::process::Command::new("rm")
-                .arg(executable.clone())
+                .arg(executable.clone() + "-args")
                 .status()
                 .unwrap();
+            if !self.interp {
```
Comment (oflatt) [inline src/util.rs:675]: why do we only remove the executable when not interpreting?
The tests should always clean up executables.

File: infra/profile.sh [hunk 2]
```diff
@@ -27,11 +27,13 @@ bench() {
...
   cargo run --release "$1" --run-mode compile-brilift -o "$PROFILE_DIR/brilift"
-  
-  # TODO: Some of the brils result in executables that seg fault, need to figure out why
-  # For now, though, we just silently ignore them
-  hyperfine --warmup 2 --export-json "$PROFILE_DIR"/brilift.json "$PROFILE_DIR/brilift" || echo "[BRILIFT] could not run $PROFILE_NAME"
+
+  # Read the args from the file
+  ARGS=$(cat $PROFILE_DIR/brilift-args)
+
+  hyperfine --warmup 2 --export-json $PROFILE_DIR/brilift.json "$PROFILE_DIR/brilift $ARGS"
```
Comment (oflatt) [inline infra/profile.sh:36]: You'll probably want to clean up the executable and argument files here
I'd say that all of these should go in a temporary directory, not the top level

**Review by oflatt — CHANGES_REQUESTED** [review]
A couple suggestions with managing files

File: src/util.rs (hunk 1 above)
Comment (ajpal) [reply src/util.rs:675]: I'm maintaining existing behavior here: https://github.com/egraphs-good/eggcc/blob/main/src/util.rs#L663

File: infra/profile.sh (hunk 2 above)
Comment (ajpal) [reply infra/profile.sh:36]: The profiler does put files in a tmp directory and clean them up itself (see lines 5-9 of this file)

**Review by oflatt — APPROVED** [review]
lgtm

**Pull request merged**